clippy_lints/src/lib.rs

```diff
@@ -461,6 +459,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry) {
         enum_variants::PUB_ENUM_VARIANT_NAMES,
         enum_variants::STUTTER,
         if_not_else::IF_NOT_ELSE,
+        indexing_slicing::INDEXING_SLICING,
```

Comment:
I think this should be in `clippy_restriction`. It is at least defined as a restriction lint in `indexing_slicing.rs:62`.